### ISSUE TITLE
Add template version messages to the rest of the product

### ISSUE THREAD

**Issue by bpmct** [coder/coder#8657]
We have template version messages, but they are not displayed everywhere yet.

- [ ] Version picker
  ![Screenshot 2023-07-21 at 11 00 02 AM](https://github.com/coder/coder/assets/22407953/12453180-12a0-412c-8c4d-e988c7bd34a7)
- [ ] VS Code extension
  ![Screenshot 2023-07-21 at 12 20 16 PM](https://github.com/coder/coder/assets/22407953/290dea6e-04b9-4d54-8c71-bf4cb75a38b8)
- [ ] Versions list in UI
  ![Screenshot 2023-07-21 at 12 21 15 PM](https://github.com/coder/coder/assets/22407953/0cc7fe28-755b-4a23-8682-2e0761599ad9)
- [ ] `coder ssh`
   ```sh
    👋 Your workspace is outdated! Update it here: https://dev.coder.com/@bpmct/dev
    ```

Anywhere else I'm missing?

**Comment by aslilac** [comment]
It looks like the VSCode extension actually does support this: https://github.com/coder/vscode-coder/blob/3c668ebf4ee8d24b2615b54efa385d17444bdf3d/src/remote.ts#L314C20-L314C20

**Comment by matifali** [comment]
@aslilac No, it does not. Here, we are talking about the changelog message. Currently, we only show if a new version is a message. We also want to display the message about what is changed in the version (if it exists). @bpmct screenshot is only to show where we can add that message.

**Comment by matifali** [comment]
Why, though I have never seen this on.

**Comment by aslilac** [comment]
Do we actually care about `coder ssh` displaying the message if it links to a page which will have the message plastered all over it? Fine if the answer is yes, just curious. (See #9705 for a screenshot)

**Comment by aslilac** [comment]
> Why, though, I have never seen this.

![Image](https://github.com/coder/coder/assets/418348/a09b0f02-f002-4c63-b05d-9d92a45193fe)

Ha, looks like it was added in the commit after the current release. We just need to publish a new version.

**Comment by aslilac** [comment]
cc @code-asher

**Comment by code-asher** [comment]
Published v0.1.22!

**Comment by aslilac** [comment]
Thank you!
